frontend/src/metabase/collections/components/CollectionHeader/tests/setup.tsx

```diff
@@ -23,15 +23,6 @@ const getProps = (
   onCreateBookmark: jest.fn(),
   onUpload: jest.fn(),
   onDeleteBookmark: jest.fn(),
-  location: {
```

Comment:
`location` was unused in `frontend/src/metabase/collections/components/CollectionHeader/CollectionHeader.tsx`